### PR TITLE
add Thunisoft full platform 8

### LINKED ISSUE #1038: Thunisoft Application Servrer V2.8 as a Compatible Software Product for Jakarta EE 8
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The URL to the Eclipse Foundation profile to confirm that you are an authorized representative of the company whose product information you are submitting**

https://www.eclipse.org/membership/showMember.php?member_id=1481

**A product name and version**

Thunisoft Application Servrer V2.8

**The Profile (Full Platform, Web Profile, Both) that the Compatible Product is submitted to be listed under**

Full Platform

**A ZIP file with a product logo with a transparent background in SVG format (optional)**

[tas-logo.zip](https://github.com/jakartaee/jakarta.ee/files/5770050/tas-logo.zip)

**The Participant, Enterprise or Strategic Member name supporting the Compatible Software Product**

Beijing Thunisoft Information Technology Co.,Ltd

**The URL to the approved certification issue demonstrating compatibility as defined in the Jakarta EE TCK Process**

https://github.com/eclipse-ee4j/jakartaee-platform/issues/290

**The URL where Compatible Software Product is available for download**

http://tas.thunisoft.com/download

### PR DESCRIPTION
**Pull request by flora8984461**
Fix #1038 

download link is updated

@chrisguindon I cannot set Chris as reviewer, at him here to review. 

Signed-off-by: Yi Liu <yi.liu@eclipse-foundation.org>

### CODE REVIEW

**Review by autumnfound — APPROVED** [review]

**Comment by zcwslnh** [comment]
> Fix #1038
> 
> Waiting for download link
> 
> Signed-off-by: Yi Liu [yi.liu@eclipse-foundation.org](mailto:yi.liu@eclipse-foundation.org)

@flora8984461 The download link has been updated to http://tas.thunisoft.com/download

File: data/compatible_products.yml [hunk 1]
```diff
@@ -134,6 +134,16 @@ sets:
...
+
+      - name: Thunisoft Application Servrer
+        vendor: Beijing Thunisoft Information Technology Co.,Ltd
+        image: '/images/compatible_products/tas-logo.png'
+        image_width: 200
+        download: 'http://www.thunisoft.com/tastckresults/index.jhtml'
+        versions:
+        - version: 2.8
+          compatibility: 'http://www.thunisoft.com/tastckresults/index.jhtml'
+          download_url: 'http://www.thunisoft.com/tastckresults/index.jhtml'
```
Comment (ivargrimstad) [inline data/compatible_products.yml:146]: Should be http://tas.thunisoft.com/download

**Comment by flora8984461** [comment]
@ivargrimstad @zcwslnh Thanks! updated.

**Comment by chrisguindon** [comment]
@TanjaObradovic can you confirm that we can push this live?

**Review by ivargrimstad — APPROVED** [review]

**Comment by chrisguindon** [comment]
Thanks @ivargrimstad

**Comment by TanjaObradovic** [comment]
Hey there!
"Thunisoft Application Servrer" - please correct the spelling "Server"

**Comment by chrisguindon** [comment]
@flora8984461 Can you  fix the typo please?

**Comment by zcwslnh** [comment]
I apologize for my mistake。I have corrected it #1038

**Comment by flora8984461** [comment]
> I apologize for my mistake。I have corrected it #1038

Thank you so much.  I did a fix in https://github.com/jakartaee/jakarta.ee/pull/1044 and it will be updated on prod soon.

**Pull request merged**